app/javascript/controllers/import_upload_controller.js

```diff
@@ -45,6 +45,7 @@ export default class extends Controller {
   }
 
   click() {
+    this.preventDefault();
     this.inputTarget.click();
   }
```

Comment:
I missed this in a prior PR, but I think rather than preventing the default behavior here, we should just rename this method to something that doesn't conflict with the global namespace per the docs:

https://stimulus.hotwired.dev/reference/actions#naming-conventions

Reply:
[c9b8a49](https://github.com/maybe-finance/maybe/pull/1111/commits/c9b8a49f1aadc057d961ff71756f8f354fff048d)

Reply:
I may be missing something here, but are we using this method anywhere now?  The import flow seems to work fine for me without this at all.